akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/MessageAdapterSpec.scala

```diff
@@ -281,7 +281,7 @@ class MessageAdapterSpec
     val deadLetterProbe = testKit.createDeadLetterProbe()
 
     val snitch = Behaviors.setup[PingReply] { context =>
-      val replyTo = context.messageAdapter[Pong](PingReply)
+      val replyTo = context.messageAdapter[Pong](PingReply.apply)
```

Comment:
can these be written as `PingReply(_)`?

Reply:
anyway, `PingReply.apply` is better than the original

Reply:
`PingReply(_)` works, I'm happy to switch to this style if preferred.

Reply:
no need to change, I think we use a `.apply` in many places already